ipaserver/install/dsinstance.py

```diff
@@ -227,7 +228,8 @@ def __init__(self, realm_name=None, domain_name=None, fstore=None,
     def __common_setup(self):
 
         self.step("creating directory server instance", self.__create_instance)
-        self.step("tune ldbm plugin", self.__tune_ldbm)
+        if get_default_db_lib == 'bdb':
```

Comment:
The lint failed because you are missing parens after get_default_db_lib. It's comparing the function to the string.

Reply:
🤦 fixed